pkg/internal/mock/didcomm/protocol/mock_didexchange.go

```diff
@@ -30,6 +30,7 @@ type MockDIDExchangeSvc struct {
 	UnregisterActionEventErr error
 	RegisterMsgEventErr      error
 	UnregisterMsgEventErr    error
+	AcceptError              error
```

Comment:
Separating AcceptError based on exchange type might help to have more control on the mock.

Reply:
But we don't need it now for current test cases, I added general mock error for all accept cases together since we test exchange/invitation separately (didn't want to have one extra field in private function of creating test handler which already has too many args).

Reply:
Ok. I was thinking about the cases where we want one accept to succeed and other fail. We might need to refactor this and other usages/consumers of this mock.